torch/fx/passes/backends/nvfuser/operator_support.py

```diff
@@ -0,0 +1,180 @@
+import typing as t
+
+import torch
+import torch.fx
+from torch.fx.passes.operator_support import OperatorSupport
+from torch.fx.passes.tools_common import CALLABLE_NODE_OPS
+
+try:
+    from torch._C._nvfuser import FusionDefinition as fd  # type: ignore[import]
+except ImportError:
+    # Not all PyTorch builds have nvfuser
+    pass
+
+class NvFuserOperatorSupport(OperatorSupport):
+    """
+    Operator support for nvFuser backend.
+
+    Note: When adding a rule, please add it to the corresponding secion and follow the
+    alphabetical order.
+    """
+
+    def __init__(self):
+        support_dict = {
+            # ===============================================================
+            # call_function aten
+            # ===============================================================
+            # Following supported aten ops is copied from torch/csrc/jit/codegen/cuda/parser.cpp
```

Comment:
That list is for TorchScript. The modern and future way of using nvFuser is through primTorch's prims (`torch.ops.prims`, not `torch.ops.aten`).

Reply:
For first prototype, we are still partitioning based on aten ops.